src/main/java/frc/team4909/robot/Robot.java

```diff
@@ -39,30 +18,30 @@
  * project.
  */
 public class Robot extends TimedRobot {
-  private static final String kDefaultAuto = "Default";
-  private static final String kCustomAuto = "My Auto";
-  private String m_autoSelected;
-  public static DriveTrainSubsystem drivetrainsub;
+  // Operator Input
+  public static BionicF310 driverGamepad;
 
+  // Subsystems
+  public static DriveTrainSubsystem drivetrainSubsystem;
   public static IntakeSubsystem intakeSubsystem;
 
-  private final SendableChooser<String> m_chooser = new SendableChooser<>();
-  public static BionicF310 driverGamepad;
-  private static Linefollow linefollow;
-
-  // LIDAR lidar1;
   /**
    * This function is run when the robot is first started up and should be used
    * for any initialization code.
    */
   @Override
   public void robotInit() {
-    m_chooser.setDefaultOption("Default Auto", kDefaultAuto);
-    m_chooser.addOption("My Auto", kCustomAuto);
-    SmartDashboard.putData("Auto choices", m_chooser);
-    driverGamepad = new BionicF310(0, 0, 0.6);
-    drivetrainsub = new DriveTrainSubsystem();
+    // Subsystems
+    drivetrainSubsystem = new DriveTrainSubsystem();
     intakeSubsystem = new IntakeSubsystem();
+
+    // Operator Input
+    driverGamepad = new BionicF310(0, 0, 0.6);
```

Comment:
I'm not entirely sure how it was selected– 0.6 is the sensitivity coefficient, it maps the linear axis to a cubic function so that drivers have more control at the center of the joystick, and speed farther away. it's likely this was just copied from last years code.

@ashwinc12 may have more input on how this was determined

Reply:
either way, I'll isolate this constants to RobotConstants and provide a more descriptive definition there

Reply:
yeah, that's what I thought, I'll fix that as mentioned earlier